autoload/youcompleteme.vim

```diff
@@ -84,6 +84,7 @@ function! youcompleteme#Enable()
     autocmd InsertLeave * call s:OnInsertLeave()
     autocmd InsertEnter * call s:OnInsertEnter()
     autocmd VimLeave * call s:OnVimLeave()
+    autocmd CompleteDone * call s:OnCompleteDone()
```

Comment:
7.3.598
I increased the global required Vim version to that, per your earlier comment.